src/types.ts

```diff
@@ -331,6 +333,8 @@ export interface LookerEmbedEventMap {
 
   'look:run:start': (this: LookerEmbedLook, event: LookEvent) => void
   'look:run:complete': (this: LookerEmbedLook, event: LookEvent) => void
+  'look:save:complete': (this: LookerEmbedLook, event: LookEvent) => void
+  'look:delete:complete': (this: LookerEmbedLook, event: LookEvent) => void
```

Comment:
If you're including the spaceId now you could extend the LookEvent type to include that? Also I wonder how we can document version availability.